lib/CMakeLists.txt

```diff
@@ -1,15 +1,11 @@
 cmake_minimum_required(VERSION 3.13)
 
 project(tree-sitter
-        VERSION "0.23.0"
+        VERSION "0.23.2"
```

Comment:
Don't forget to bump this too. ;)

Reply:
Don't forget that the version _on this branch_ is (or should be) 0.24.0 -- point releases only come from the release-0.23 branch.